src/tileset.rs

```diff
@@ -17,6 +17,10 @@ pub use wangset::*;
 /// Also see the [TMX docs](https://doc.mapeditor.org/en/stable/reference/tmx-map-format/#tileset).
 #[derive(Debug, PartialEq, Clone)]
 pub struct Tileset {
+    /// The path first used in a [`ResourceReader`] to load this tileset.
+    ///
+    /// For embedded tilesets, this path will be the same as the template or map's source.
```

Comment:
That's an interesting choice. I might have made it an `Option<PathBuf>` and only set it for non-embedded tilesets. But maybe storing the original file path is actually more useful.

Reply:
Yeah, thought about it, but storing the original source looks pretty useful. It's true that it's harder to know whether it's embedded or not but that's kinda an internal detail anyway